spec/researches/getPassiveVoiceFrenchSpec.js

```diff
@@ -0,0 +1,83 @@
+var passiveVoice = require( "../../js/researches/getPassiveVoice.js" );
+var Paper = require( "../../js/values/Paper.js" );
+
+describe( "detecting passive voice in sentences", function() {
+
+	it( "returns active voice (présent)", function () {
+		var paper = new Paper("Je mange une pomme." , { locale: "fr_FR" } );
```

Comment:
Please check spacing:
- Add spaces within parentheses
- Remove spaces after the `function` keword and before commas